tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,7 @@
+Exit Code: 1
+Standard output:
+lib/NormalModule.js(163,12): error TS2451: Cannot redeclare block-scoped variable 'module'.
```

Comment:
webpack is creating its own `module` a la `const module = new NativeModule(filename, this); return module.exports`. This *already* has some suppressed errors, although this one is not a good one since the global `module` is not supposed to be a block-scoped variable. I think some predicate I recently added needs to check SymbolFlags.ModuleExports in addition to looking for the syntax `module.exports`.